carAPI/src/main/java/com/everis/control/CarService.java

```diff
@@ -67,7 +67,12 @@ public Car updateCar(Car car) {
 	public Car deleteCar(long id) {
 		LOG.info("Deleting car by id: " + id);
 		Car car = getCar(id);
-		em.remove(car);
-		return car;
+		if(car == null) {
+			return null;
```

Comment:
I think it would be better to return NOT_FOUND or some status to the user, otherwise it wont never know what happened, whether the car was deleted or not, or found or not ..... 
Please check this : https://www.restapitutorial.com/lessons/httpmethods.html